cmd/nerdctl/run_windows_test.go

```diff
@@ -65,3 +68,42 @@ func TestRunProcessIsolated(t *testing.T) {
 	containerUser := "ContainerUser"
 	base.Cmd("run", "--rm", "--isolation=process", "-u", containerUser, testutil.WindowsNano, "whoami").AssertOutContains(containerUser)
 }
+
+func TestRunHyperVContainer(t *testing.T) {
+	testutil.DockerIncompatible(t)
+	base := testutil.NewBase(t)
+	hypervServiceName := "vmcompute"
+
+	m, err := mgr.Connect()
+	if err != nil {
+		t.Fatalf("unable to access Windows service control manager: %s", err)
+	}
+	defer m.Disconnect()
+
+	s, err := m.OpenService(hypervServiceName)
```

Comment:
I think (emphasis on think) vmcompute should exist with just the containers feature on. Don't have a windows machine for a little bit but I can verify

Reply:
(if so) we'd need a better check

Reply:
This is correct @dcantah, the service is present with only the containers feature on.

Reply:
There is another service called "vmms", will parse for that.